modules/katamari/src/main/ts/ephox/katamari/api/Thunk.ts

```diff
@@ -1,7 +1,7 @@
-export const cached = function (f: Function) {
+export const cached = <T extends (...args: any[]) => any>(f: T) => {
```

Comment:
We weren't preserving types for functions being passed in, so this should preserve the types now.

Reply:
Yeah, that'd make sense 😄